src/components/FCard.vue

```diff
@@ -36,15 +49,19 @@ export interface FCardProps {
 
 const props = withDefaults(defineProps<FCardProps>(), {
   color: 'primary',
+  hoverColor: 'primary',
   textColor: 'neutral--light-5',
+  hoverTextColor: 'neutral--light-5',
```

Comment:
With that, when a user sets `color` and `textColor` but *do not set* the hover ones, we will **always** have a hover animation with the defaults colors but he does not want it !

What I would do is that the _hover color_ and _hover text color_ both have a default to `undefined`, and you make a computed:
```ts
const classes = computed(() => ({
  'FCard--hoverAnimated': !!props.hoverColor || !!props.hoverTextColor,
});
```
this adds the class to the card only if needed, and this class defines the `:hover` CSS state.